services/object/src/backends/test.js

```diff
@@ -4,6 +4,14 @@ const { Backend } = require('./base');
  * The test backend type is only available when running tests.
  */
 class TestBackend extends Backend {
+  getDetails(name) {
```

Comment:
This method name seems pretty vague.  And, it should be defined in the `Backend` class with an explanation of its meaning.  Also, it looks like the test implementation here has a different set of arguments (just name) to the way it's called in `api.js`.